code/modules/surgery/xeno.dm

```diff
@@ -70,7 +70,7 @@
 			SPAN_NOTICE("[user] successfully destroys your carapace into bits and pieces using [tool]."),,
 			SPAN_NOTICE("[user] successfully destroys [target.caste_type] carapace into bits and pieces using [tool]."))
 	for(var/mob/living/carbon/human/victim in orange(1, target))
-		if(istype(victim.wear_suit, /obj/item/clothing/suit/bio_suit) && istype(victim.head, /obj/item/clothing/head/bio_hood))
+		if((istype(victim.wear_suit, /obj/item/clothing/suit/bio_suit) || istype(victim.wear_suit, /obj/item/clothing/suit/storage/synthbio)) && (istype(victim.head, /obj/item/clothing/head/bio_hood) || istype(victim.head, /obj/item/clothing/head/bio_hood/synth)))
```

Comment:
Turn this into a trait on clothing or something similar, this is gross

Reply:
k will do